internal/jobcontainers/path.go

```diff
@@ -156,7 +152,7 @@ func getApplicationName(commandLine, workingDirectory, pathEnv string) (string,
 	if quoteCmdLine {
 		trialName = "\"" + trialName + "\""
 		trialName += " " + strings.Join(args[argsIndex+1:], " ")
-		adjustedCommandLine = trialName
+		adjustedCommandLine = trialName[0 : len(trialName)-1] // Take off the last empty space from above when joining the args.
```

Comment:
Would https://pkg.go.dev/strings#TrimSpace this work instead?

Reply:
Oh sweet, forgot there was something that removed whitespace at the beginning/end.